src/Factory.php

```diff
@@ -107,7 +107,7 @@ public function create($spec): ElementInterface
         $spec = $this->validateSpecification($spec, __METHOD__);
         $type = $spec['type'] ?? Element::class;
 
-        $element = $this->getFormElementManager()->get($type);
+        $element = $this->getFormElementManager()->get($type, $spec['options'] ?? []);
```

Comment:
Build parameters must be `['options' => $spec['options']]` according to ElementFactory and if options are not set or empty build parameters should be null to not trigger plugin manager's `build()` behavior.

ElementFactory also accepts element name as build parameters, so we should probably pass it too if `$spec` has it.

Something along those lines:

```suggestion
        $buildParams = null;
        if (! empty($spec['options'])) {
            $buildParams = ['options' => $spec['options']];
            $name = $spec['name'] ?? null;
            if ($name !== null && $name !== '') {
                $buildParams['name'] = $name;
            }
        }
        $element = $this->getFormElementManager()->get($type, $buildParams);
```

Reply:
note that options could be iterable, according to code. I did not handle it in suggested change.